assets/onboarding/query-string-router/url-functions.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Add param to the URL with pushState.
+ *
+ * @param {string}  paramName  Param name to be added to the URL.
+ * @param {string}  paramValue Param value to be added to the URL.
+ * @param {boolean} replace    Flag if it should replace the state. Otherwise it'll push a new.
+ */
+const pushQueryStringState = ( paramName, paramValue, replace = false ) => {
```

Comment:
Do we need to worry about browser compatibility for this?

Reply:
I don't think so. The history API with the `replaceState` and `pushState` are present since IE10 \o/